index.bs

```diff
@@ -213,13 +214,18 @@ Session configuration {#session-configuration}
 dictionary XRDepthStateInit {
   required sequence<XRDepthUsage> usagePreference;
   required sequence<XRDepthDataFormat> dataFormatPreference;
+  boolean matchDepthView = true;
```

Comment:
Github won't seem to let me add a comment there, but does https://github.com/immersive-web/depth-sensing/pull/50/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985L649 need to be updated as well? (The first part of "Native Device Concepts" talks about depth MUST be View aligned).